### PR TITLE
add customer list and add customer form

### PR DESCRIPTION
**Pull request by dlimx**
Screen Shot 2020-07-20 at 17 22 40
Screen Shot 2020-07-20 at 17 22 44

### CODE REVIEW

File: src/components/CustomerAdd/CustomerAdd.jsx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+  const validate = (data) => {
+    const schema = yup.object().shape({
+      first_name: yup.string().required(),
+      last_name: yup.string().required(),
+      birthdate: yup.date().required(),
+      address: yup.string(),
+      city: yup.string(),
+      zip: yup.string(),
+      state: yup.string(),
+      image: yup.string(),
```
Comment (blakelester) [inline src/components/CustomerAdd/CustomerAdd.jsx:29]: I'm good with what you have as required here, I can update our DB outline to allow NULL. This is the current specs:

Customers: will keep track of the customers that buy our products
id: int, auto increment, unique, not NULL, PK 
first_name: varchar(255), not NULL
last_name: varchar(255), not NULL
birthdate: date
address: varchar(255), not NULL
city: varchar(255), not NULL
state: varchar(255), not NULL
zip: varchar(255), not NULL
created_at: date, not NULL
updated_at: date, not NULL
image: varchar(255)

File: src/components/Navbar/Navbar.jsx [hunk 2]
```diff
@@ -19,12 +19,18 @@ export default function Navbar() {
         <NavLink className="Navbar" activeClassName="NavbarActive" exact to="/">
           Products
         </NavLink>
+        <NavLink className="Navbar" activeClassName="NavbarActive" to="/filter">
+          Filters
+        </NavLink>
+        <NavLink className="Navbar" activeClassName="NavbarActive" to="/order">
+          History
```
Comment (blakelester) [inline src/components/Navbar/Navbar.jsx]:
```suggestion
          Orders
```

File: src/components/CustomerAdd/CustomerAdd.jsx (hunk 1 above)
Comment (dlimx) [reply src/components/CustomerAdd/CustomerAdd.jsx:29]: I think `birthdate` can be optional too?

File: src/components/Navbar/Navbar.jsx (hunk 2 above)
Comment (dlimx) [reply src/components/Navbar/Navbar.jsx]: Done~

File: src/components/CustomerAdd/CustomerAdd.jsx (hunk 1 above)
Comment (blakelester) [reply src/components/CustomerAdd/CustomerAdd.jsx:29]: Good with me @dlimx

**Review by blakelester — APPROVED** [review]

**Pull request merged**